### PR TITLE
rtmp-services: add Odysee.com

### PR DESCRIPTION
**Pull request by iamflorencejay**
### Description

This RMTP update is to add Odysee on the rmtp services list.
### Motivation and Context

Some Odyseenauts are not fixing the 2500kpbs bitrate issue, so I've added the recommended settings on this code.

### How Has This Been Tested?

This is tested using Manjaro on my Lenovo Laptop. And it is flawless.

### Types of changes

Added RMTP services

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [ ] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

**Comment by WizardCM** [comment]
Please fill in the full PR template.

**Comment by iamflorencejay** [comment]
Copy thar

**Comment by iamflorencejay** [comment]
@WizardCM done 👍🏻

**Comment by RytoEX** [comment]
Are you an official representative for Odysee?

**Comment by iamflorencejay** [comment]
No. Just an active contributor for them. @RytoEX

File: plugins/rtmp-services/data/services.json [hunk 1]
```diff
@@ -2038,6 +2038,22 @@
...
+                {
+                    "name": "Default",
+                    "url": "rtmp://stream.odysee.com/live"
+                }
+            ],
+            "recommended": {
+                "keyint": 1,
+                "max video bitrate": 2500,
+                "profile": "high",
+                "x264opts": "tune=zerolatency"
```
Comment (notr1ch) [inline plugins/rtmp-services/data/services.json:2053]: This destroys quality for a handful of frames gained, are you sure you want this?

**Comment by iamflorencejay** [comment]
@notr1ch it is on Odysee's recommendation. You can see this on https://odysee.com/$/livestream

**Comment by iamflorencejay** [comment]
Screenshot_2021-04-29-21-39-33-962_com brave browser
Here is the screenshot @notr1ch

**Comment by notr1ch** [comment]
Seems like yet another service that doesn't know what they are doing...

I also noticed the commit and PR prefix is typoed, "rmtp" instead of "rtmp".

**Comment by iamflorencejay** [comment]
@notr1ch i think it is on beta, so it will fix soon

**Comment by RytoEX** [comment]
It seems this was merged by accident.  At this time, we require that additions to rtmp-services be made by official representatives from the service being added.  As such, this merge has been reverted.  Thank you for understanding.

**Comment by trymeouteh** [comment]
Will this also allow one to livestream on LBRY also?

**Pull request merged**